hasher-matcher-actioner/src/OpenMediaMatch/tests/test_api.py

```diff
@@ -221,6 +223,26 @@ def test_banks_add_hash_index(app: Flask, client: FlaskClient):
     assert post_response.json == {"matches": [2]}
 
 
+def test_lookup_add_hash_without_role(app: Flask, client: FlaskClient):
```

Comment:
nit: I don't think we're "adding" a hash in this unittest.  In other tests, this is referring to banking functionality.